### PR TITLE
minor documentation change regarding throttle 

### PR DESCRIPTION
**Pull request by ArbitRandomUser**
Minor change in training.md 
throttled function is defined prior to the train! function and not defined as an argument of train!
added @epoch to demonstrate typical training loop

### CODE REVIEW

File: docs/src/training/training.md [hunk 1]
```diff
@@ -117,9 +117,9 @@ A more typical callback might look like this:
 ```julia
 test_x, test_y = # ... create single batch of test data ...
 evalcb() = @show(loss(test_x, test_y))
-
-Flux.train!(objective, ps, data, opt,
-            cb = throttle(evalcb, 5))
+throttled_cb = throttle(evalcb,5)
```
Comment (ToucheSir) [inline docs/src/training/training.md:120]:
```suggestion
throttled_cb = throttle(evalcb, 5)
```

File: docs/src/training/training.md [hunk 2]
```diff
@@ -117,9 +117,9 @@ A more typical callback might look like this:
...
+@epochs 20 Flux.train!(objective, ps, data, opt,
+            cb = throttled_cb)
```
Comment (ToucheSir) [inline docs/src/training/training.md:122]:
```suggestion
Flux.@epochs 20 Flux.train!(objective, ps, data, opt, cb = throttled_cb)
```

**Review by ToucheSir — CHANGES_REQUESTED** [review]
LGTM, just added a couple of formatting nits.

**Review by ToucheSir — APPROVED** [review]
